about/people.yml

```diff
@@ -1,177 +1,109 @@
-    bio: August obtained their PhD from Brown in 2017 in Applied Mathematics and Computational
-      Biology. Outside of work they organize against criminalization and the police
-      state. Both their research practices and organizing work are guided by the belief
-      that crisis and uncertainty are a gift and that we must be involved in shaping
-      how change happens.
-    image: august_main.jpg
-  - name: Andras Zsom
+    team: Leadership
+    title: Assistant Chief Information Officer, Infrastructure and Research Computing
+    github_username: ''
+    brown_directory_uuid: 3ddfe2bd-672a-0f09-03bc-cbb37339a434
+    bio: ''
+    image: linnea_main.jpg
+  - name: Mete Tunca
     type: Full Time
-    team: Advanced Research Computing
-    subteam: Institutional Data
-    title: Lead Data Scientist
-    github_username: azsom
-    brown_directory_uuid: 84734442-fa19-4e15-8b06-9f71351e9104
-    bio: Andras is passionate about using machine learning and predictive modeling for
-      good. He has a PhD in astrophysics and he was a postdoctoral researcher at MIT
-      before coming to Brown.
-    image: andras_main.jpg
-  - name: Ashley S. Lee
+    team: Leadership
+    title: Assistant Director of Research Systems and Services
+    github_username: mttunca
+    brown_directory_uuid: d0e7b3b6-f6b4-f28b-3bfd-08bccf05ac31
+    bio: ''
+    image: mete_main.jpg
+  - name: Paul Stey
     type: Full Time
-    team: Advanced Research Computing
-    subteam: Institutional Data
-    title: Senior Data Scientist
-    github_username: alee35
-    brown_directory_uuid: 380094e2-6f9d-47cf-91ce-e636876a58cf
-    bio: Ashley received training in Biology and Data Science at the University of Virginia
-      and dabbled in a variety of careers including Bike Salesman, Bus Driver, Bagel Slinger, and Chief
-      Fiduciary at Smoothie King. She enjoys food and beverage, bodies of water, jokes, and
-      caring for her beloved dog and plants.
-    image: ashley_main.jpg
+    team: Leadership
+    title: Associate Director of Research Systems and Services
+    github_username: paulstey
+    brown_directory_uuid: 0525f19f-d156-4c47-a2e1-8b4e904a2e84
+    bio: Paul is a data scientist interested in using data and machine learning to solve
+      challenging problems. Before coming to Brown he was a statistician at the EPA.
+      He has a PhD in Developmental Psychology.
+    image: stey_main.jpg
+  - name: Sam Fulcomer
+    type: Full Time
+    team: Leadership
+    title: Associate Director
+    github_username: ''
+    brown_directory_uuid: 0d83d4b5-85a5-62fc-44a3-81eaff49e183
+    bio: ''
+    image: fulcomer_main.jpg
   - name: Thomas Serre
     type: Full Time
     team: Leadership
-    title: Faculty Director
+    title: Associate Director of the Center for Computational Brain Science, Director for the Center for Computation and Visualization, Professor of Cognitive, Linguistic and Psychological Sciences, Professor of Computer Science
```

Comment:
did this get even longer? I'm not sure this will fit on the card

Reply:
It expands? Seems alright?

<img width="430" alt="Screen Shot 2022-01-21 at 4 58 59 PM" src="https://user-images.githubusercontent.com/5270855/150605729-e2ce08d0-43da-4fa6-a3ab-d275033552ff.png">

Reply:
@paulstey would it be okay to just stick with `Faculty Director` here? Or are all titles needed? Since this is ccv's people page I lean towards only listing the role/title as it relates to ccv

Reply:
Change to just Faculty Director

Reply:
Fixed !

Reply:
I would say we should at least call out the fact that he's a full professor  (in addition to faculty director). What about something like this: `Faculty Director, Professor of Cognitive, Linguistic, and Psychological Science`?

Reply:
Fixed !

Reply:
Thank you, @eldu!!